scripts/run_installer.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -o errexit
+set -o nounset
+
+GPU_INSTALLER_ENV_KEY="cos-gpu-installer-env"
+GPU_INSTALLER_ENV_PATH="/etc/gpu-installer-env"
+
+setup() {
+  /usr/share/google/get_metadata_value "attributes/${GPU_INSTALLER_ENV_KEY}" \
+    > "${GPU_INSTALLER_ENV_PATH}"
+  source "${GPU_INSTALLER_ENV_PATH}"
+  mkdir -p "${NVIDIA_INSTALL_DIR_HOST}"
+  # Bind mount NVIDIA_INSTALL_DIR_HOST to itself to support remount.
```

Comment:
Comment should say something like: "Make NVIDIA_INSTALL_DIR_HOST executable by bind mounting it."
Any idea why this is needed though?

Reply:
NVIDIA_INSTALL_DIR_HOST needs to be executable in order to run binaries such as  NVIDIA_INSTALL_DIR_HOST/bin/nvidia-smi.